src/main/resources/db/migration/v0.34.0.3__gene_expression_annotation.sql

```diff
@@ -0,0 +1,73 @@
+CREATE SEQUENCE public.expressionannotation_seq         START WITH 1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
```

Comment:
You need to rename this file - version number clashes with latest migrations

Reply:
fixed